lib/widgets/breez_avatar_dialog.dart

```diff
@@ -195,35 +188,25 @@ Widget breezAvatarDialog(BuildContext context, UserProfileBloc userBloc) {
               child: Text('SAVE',
                   style: Theme.of(context).primaryTextTheme.button),
               onPressed: () async {
-                if (_pickedImage != null) {
-                  await _pickedImage
-                      .readAsBytes()
-                      .then(scaleAndFormatPNG)
-                      .then((imageBytes) async {
-                    var action = UploadProfilePicture(imageBytes);
-                    userBloc.userActionsSink.add(action);
-                    setState(() {
-                      _state = UPLOAD_STATE.UPLOAD_IN_PROGRESS;
-                    });
-                    await action.future.then((_) {
-                      setState(() {
-                        _state = null;
-                      });
-                    }, onError: (error) {
-                      setState(() {
-                        _pickedImage = null;
-                        _state = null;
-                      });
-                      return showFlushbar(
+                setState(() {
+                  _isUploading = true;
+                });
+                await _uploadImage(_pickedImage, userBloc).then((uploaded) {
```

Comment:
We don't need the await here.

Reply:
[Removed unnecessary async/await keyword](https://github.com/breez/breezmobile/pull/283/commits/3614a9bb9cc19102f816de2947ca1c160f674a18)